client/checkout/classic/upe.js

```diff
@@ -263,6 +263,31 @@ jQuery( function ( $ ) {
 		};
 	};
 
+	const enableStripeLinkPaymentMethod = () => {
+		const linkAutofill = api.getStripe().linkAutofillModal( elements );
+
+		document
```

Comment:
Nitpick: Would be nice to have this code in jQuery to keep it consistent with most of the code in this file. But that's only a minor nitpick. Feel free to ignore.